### PR TITLE
Implemented correct mcq past z

### PR DESCRIPTION
**Pull request by Vismai-Khanderao**
This should fix issue #3133 

Loops past `z` to `aa`, works for arbitrary number of options, e.g `.... -> zzzy -> zzzz -> aaaaa -> aaaab -> ....`

### CODE REVIEW

**Review by tdy — COMMENTED** [review]
Thanks for the PR! The linter is showing some code style issues. Would you mind cleaning these up: https://github.com/PrairieLearn/PrairieLearn/runs/1196461528

**Comment by Vismai-Khanderao** [comment]
@tdy Apologies! I had some issues running PL locally, just fixed them

**Comment by mwest1066** [comment]
Looks great! Thanks for the PR! Would you be able to make the following small changes before we merge it?

* [ ] put the new code in a helper function (perhaps `index2mc()`?) in the [`prairielearn.py` library file](https://github.com/PrairieLearn/PrairieLearn/blob/master/question-servers/freeformPythonLib/prairielearn.py)
* [ ] Use the new helper function like `keyed_answer = {'key': pl.index2mc(i), 'html': html}`
* [ ] Make sure the new helper function has a clear doc-string explaining it's purpose and usage.
* [ ] Also use this new helper function in these other locations:
https://github.com/PrairieLearn/PrairieLearn/blob/0721bbb6d3b1496f24959db69a7c156886cad39e/elements/pl-multiple-choice/pl-multiple-choice.py#L350
https://github.com/PrairieLearn/PrairieLearn/blob/0721bbb6d3b1496f24959db69a7c156886cad39e/elements/pl-checkbox/pl-checkbox.py#L94
https://github.com/PrairieLearn/PrairieLearn/blob/0721bbb6d3b1496f24959db69a7c156886cad39e/elements/pl-checkbox/pl-checkbox.py#L372

**Comment by Vismai-Khanderao** [comment]
@mwest1066 added those changes, any other ones required?

File: question-servers/freeformPythonLib/prairielearn.py [hunk 1]
```diff
@@ -1084,3 +1084,25 @@ def escape_invalid_string(string):
     Wraps and escapes string in <code> tags.
     """
     return f'<code class="user-output-invalid">{html.escape(escape_unicode_string(string))}</code>'
+
+def index2key(i):
+    """
+    index2key(i)
+
+    Used when generating alphabet ordered lists.
```
Comment (mwest1066) [inline question-servers/freeformPythonLib/prairielearn.py:1093]:
```suggestion
    Used when generating ordered lists of the form ['a', 'b', ..., 'z', 'aa', 'ab', ..., 'zz', 'aaa', 'aab', ...]
```

File: question-servers/freeformPythonLib/prairielearn.py [hunk 2]
```diff
@@ -1084,3 +1084,25 @@ def escape_invalid_string(string):
...
+
+    Returns alphabetic key in the form [a-z]* from given integer (i) representing alphabet index (base-0).
```
Comment (mwest1066) [inline question-servers/freeformPythonLib/prairielearn.py:1095]:
```suggestion
    Returns alphabetic key in the form [a-z]* from a given integer (i = 0, 1, 2, ...).
```

**Comment by mwest1066** [comment]
> @mwest1066 added those changes, any other ones required?

This is excellent, thanks!

**Pull request merged**